openpype/hosts/maya/plugins/publish/validate_model_content.py

```diff
@@ -59,12 +61,13 @@ def get_invalid(cls, instance):
 
         # Top group
         assemblies = cmds.ls(content_instance, assemblies=True, long=True)
-        if len(assemblies) != 1:
+        if len(assemblies) != 1 and cls.validate_top_group:
             cls.log.error("Must have exactly one top group")
-            if len(assemblies) == 0:
-                cls.log.warning("No top group found. "
-                                "(Are there objects in the instance?"
-                                " Or is it parented in another group?)")
+            return assemblies
+        if len(assemblies) == 0:
```

Comment:
I think this condition should be first otherwise will never happen.

Reply:
I believe so. Even if you don't validate single top group, it should never be 0. That would mean empty publish